blocks_vertical/data.js

```diff
@@ -544,7 +545,7 @@ Blockly.Constants.Data.RENAME_OPTION_CALLBACK_FACTORY = function(block) {
 Blockly.Constants.Data.DELETE_OPTION_CALLBACK_FACTORY = function(block) {
   return function() {
     var workspace = block.workspace;
-    var name = block.getField('VARIABLE').text_;
```

Comment:
We probably want this change above in RENAME_OPTION_CALLBACK_FACTORY also?

Not sure if this is the cause, but it seems that renaming doesn't work for the variable getter blocks, but does work for the variable dropdown blocks.

Reply:
Fixed the renaming problem. This wasn't the issue, but I changed it to match DEFAULT_OPTION_CALLBACK_FACTORY for good measure.